src/reporting/clients/AggregationsClient.ts

```diff
@@ -0,0 +1,350 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type { AccessToken } from "@itwin/core-bentley";
+import { RequiredError } from "../interfaces/Errors";
+import type { EntityListIterator } from "../../common/iterators/EntityListIterator";
+import { EntityListIteratorImpl } from "../../common/iterators/EntityListIteratorImpl";
+import { Collection, getEntityCollectionPage } from "../../common/iterators/IteratorUtil";
+import { OperationsBase } from "../../common/OperationsBase";
+import { AggregationProperty, AggregationPropertyCollection, AggregationPropertyType, AggregationPropertySingle, AggregationPropertyUpdate, AggregationPropertyCreate,
```

Comment:
I think it would be good to follow the itwinjs-core formatting standards here. Please look at this file: https://github.com/iTwin/itwinjs-core/blob/master/core/backend/src/IModelDb.ts